index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/1bc3a7e061.js" crossorigin="anonymous"></script>
+    <title>Mobile first Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <nav class="nav-menu flex">
+            <div class="logo flex">
+                <img src="./Images/Logo.png" alt="Logo">
+                <p>My Portfolio</p>
+            </div>
+            <div class="hamburger-menu flex">
+                <a href="#">
+                    <i class="fa-solid fa-bars"></i>
+                </a>
+            </div>
+        </nav>
+    </header>
+    <main class="main">
+        <section class="headline flex">
+            <h2 class="headline-text">I’m Belay. <br> <span>I’m a software developer</span> </h2>
+            <p class="description-text"> I can help you build a product , feature or website
+                Look through some of my work and experience! If you
+                like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+            <div>
+                <p class="social-icons-header">Let's Connect</p>

```

Comment:
- [ ] [REQUIRED] Please change the `p` tag to `h1` tag because `h1` is more semantic 😄